hail/python/requirements.txt

```diff
@@ -1,3 +1,4 @@
+aiohttp
```

Comment:
From https://packaging.python.org/discussions/install-requires-vs-requirements/#install-requires:

"It is not considered best practice to use install_requires to pin dependencies to specific versions, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades."

I don't think we should be (semi) pinning dependencies here unless (1) we require a minimum version, or (2) there are specific versions we know don't work.  The one exception I'm aware of is pyspark, which we need to specify precisely for jar compatibility.

@tpoterba does that sound right?  In which case, I'm going to unpin these packages.